### DISCUSSION TITLE
How to transform seurat object into archr?

### DISCUSSION THREAD

**Discussion by wanglu2014** [GreenleafLab/ArchR#1956] [Questions / Documentation]
For example the All-snATAC-Spatial multi-omic map of human myocardial infarction oject in https://cellxgene.cziscience.com/collections/8191c283-0816-424b-9b61-c3e1d6258a77
![图片](https://github.com/GreenleafLab/ArchR/assets/12541282/08c96a12-8652-48e6-94e4-00e386f86477)

**Comment by rcorces** [comment]
this is not possible
